client/src/Home.jsx

```diff
@@ -2,16 +2,41 @@ import './App.css'
 import 'bootstrap/dist/css/bootstrap.min.css';
 import MyNavBar from './components/Navbar'
 import FilmList from './components/FilmList'
-
+import FilmDetail from './components/FilmDetail';
+import ViewFilm from './components/ViewFilm';
+import {Link,Routes, Route } from 'react-router-dom';
+import Welcome from './components/Welcome';
+import form from './components/Form';
```

Comment:
ESLint was bugging me about your file path here. Since the file name is lowercase, have its reference here be lowercase, too